pycloudlib/lxd/instance.py

```diff
@@ -351,11 +351,10 @@ def _wait_for_cloudinit(self, *, raise_on_failure: bool):
                         continue
 
                     # We are getting those types of messages when booting
-                    # lxd vms. However, if we wait a moment before trying
+                    # lxd images. However, if we wait a moment before trying
                     # again, the error fades away. That's why we are treating
                     # it here.
-                    print("cloud-init failed to start: out: ......." in str(e))
-                    if "cloud-init failed to start: out: ......." in str(e):
+                    if "cloud-init failed to start: out:" in str(e):
                         time.sleep(sleep_time)
```

Comment:
We probably need to log a debug level message here. ("Retrying on expected failure: %s", str(e)) so we can grok what is going on as this has a potential to keep retrying for other potentially unexpected failure cases where we should exit right?